design/baremetal-operator/user-defined-root-device-hints.md

```diff
@@ -13,7 +13,7 @@ implemented
 
 ## Summary
 
-This document explains how rootdevicehints could be given as user
+This document explains how `rootdevicehints` could be given as user
```

Comment:
nit: `rootDeviceHints` to match the actual capitalization in the JSON body

Reply:
I'll fix that.